lib/cfg-grammar.y

```diff
@@ -330,6 +330,9 @@ extern struct _StatsOptions *last_stats_options;
 
 %token KW_RETRIES                     10511
 
+/* authentication */
+%token KW_AUTH                        10600
```

Comment:
You can declare this token in `redis-grammar.ym` if you don't want to use it in multiple modules.